bedrock/base/templates/base-pebbles.html

```diff
@@ -102,9 +102,16 @@
             <li class="first about-item">
               <a href="{{ url('mozorg.about') }}" data-link-type="nav" data-link-name="About">{{_('About')}}</a>
             </li>
+            {% if LANG.startswith('en-') %}
```

Comment:
We can't use an l10n tag in the base templates, so we're doing `en` only for now until enough locales translate the strings.